resources/styles/book-content/ap-bio/ap-connection.less

```diff
@@ -0,0 +1,71 @@
+.ap-connection{
+   h1 + p:first-of-type{
+  &::first-letter{
+    float: left;
+    #fonts > .sans(1.75rem, 3.15rem); //setting line-height in em to scale on browser zoom
+    font-weight: 400;
+    color: @tutor-neutral-dark;
+    margin: .0; //best combination on line-height and margin to look good in chrome and ff
+  }
+  // ensure that the p is heigh enough for the drop cap
+  // to fit beside it, otherwise it will overlay on top of the next element
+  min-height: auto;
+}
+  background: @tutor-neutral-lightest;
+  margin: 40px 0 32px 0;
+  clear: both;
+  border-top: solid 8px @tutor-neutral-lighter;
+  border-bottom: solid 8px @tutor-neutral-lighter;
+  padding: 20px 40px;
+  p {
+    #fonts > .sans(1.8rem, 1.75em);
+    color: @tutor-neutral-dark;
+    }
+  &:before {
+    display: none;
+    position: absolute;
+    //content: attr(data-type);
+    #fonts > .sans(@tutor-book-ui-top-height - 18px, @tutor-book-ui-top-height);
+    font-weight: 900;
+    color: @tutor-book-ap-bio-primary;
+    background: @tutor-book-ap-bio-secondary;
+    padding: 0 40px;
+    height: @tutor-book-ui-top-height;
+    display: inline-block;
+    margin: -68px 0 0 -40px;
```

Comment:
If these are based on other margins can you include a variable (in `variables.less` if it is used in multiple files or prefix the var name with `book-bio-ap-connections-` if it is local (the name is based on the path so it does not conflict with other "local" vars)?